tests/symbolic.py

```diff
@@ -146,5 +149,28 @@ def test_shred(self):
         self.assertEqual(ezero, ezero.shred())
 
 
+class TestRepresentative(unittest.TestCase):
+    def test_expected_error(self):
+        a = Var("a")
+        b = Var("b")
+        x = Var("x")
+        y = Var("y")
+        z = Var("z")
+
+        def R(*args): return Representative([*args])
+
+        e1 = R(x, y, z)
+        self.assertAlmostEqual(1/3, e1.expected_error(x), delta=10e-5)
```

Comment:
You mean 1e-5 (meaning 1*10^-5) here. Can you factor this delta out and add some thoughts on why we don't expect the precise fractions here? Is it possible this algorithm works with Python's Fraction class?

Reply:
Should work with fractions. I'll try.